File: SignalRChat/scripts/Models/Message.js
(function () {

    'use strict';

    
    define(function () {

        function Message() {

        }

        Message.create = function (message) {
            if (message !== null && typeof message === 'object') {
                return Object.assign(new this(), message);
            }
            return new this();
        };

        Message.prototype.UserName = "";
        Message.prototype.Content = "";
        Message.prototype.SendTime = null;

        
        return Message;

    });

})();

File: SignalRChat/Models/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SignalRChat.Models
{
    public class Message
    {
        public string UserName { get; set; }
        public string Content { get; set; }
        public string SendTime { get; set; }
    }
}